src/main/java/COSE/ECPrivateKey.java

```diff
@@ -45,7 +45,7 @@ public ECPrivateKey(OneKey oneKey) throws CoseException, IOException
             System.arraycopy(X, 0, rgb, 1, X.length);
             rgb[0] = (byte) (2 + (oneKey.get(KeyKeys.EC2_Y).AsBoolean() ? 1 : 0));
             pubPoint = p.getCurve().decodePoint(rgb);
-            point = new ECPoint(point.getAffineX(), point.getAffineY());
+            point = new ECPoint(pubPoint.getAffineXCoord().toBigInteger(), pubPoint.getAffineYCoord().toBigInteger());
```

Comment:
1) I'm not sure if it's an absolute value or not.  I will have to check, but it might be possible for it to be negative.
2) This is importing from a transport value into the mathematical concepts, so bitsize doesn't matter here.

One thing to consider is this **should** be temporary.  Once all the rest of the library is using JavaCE, this class becomes irrelevant (I think).  Assuming it is, I'm not sure it's worth much analysis right now; if not, then I'll definitely make sure the questions are resolved.